### PR TITLE
Enable GPS Rescue UI if used as a mode

### LINKED ISSUE #3458: Enable GPS rescue settings when GPS rescue is not selected as stage2 failsafe
### Is your feature request related to a problem? Please describe

I wanted to configure GPS rescue on a switch, but not enable it as my stage2 failsafe. After looking at the code and testing it, I found that this is a valid and supported configuration. However, the configurator will not allow GPS rescue settings to be changed unless GPS rescue is selected as the stage2 failsafe.

It is inconvenient and confusing to have to enable GPS stage2 failsafe in order to change these settings.

In my opinion, GPS rescue is a separate feature from failsafe, and should not be conflated in the way it is at present.

### Describe the solution you'd like

I would suggest that the selection of a stage2 failsafe be decoupled from the GPS rescue settings, thus allowing GPS rescue to be configured even if it is not selected as the stage2 failsafe behaviour.

### Describe alternatives you've considered

These options could be moved to a new location, and only enabled if either stage2 failsafe GPS rescue, or mode based GPS rescue is enabled. However, I see no reason to disable them in any situation.

### PR DESCRIPTION
**Pull request by atomgomba**
Fixes #3458. Currently the UI for changing GPS Rescue settings is only enabled if the failsafe stage 2 is configured as GPS Rescue. Because the GPSR settings can be changed at anytime using the CLI, it makes sense to enable the UI also if GPSR is enabled as a mode to make things easier for users.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by nerdCopter — APPROVED** [review]
* approving untested

**Pull request merged**
